src/components/IssueEditOld.vue

```diff
@@ -0,0 +1,212 @@
+<script setup lang="ts">
```

Comment:
Same here, another old vs amending IssueEdit directly. What's the reason for this?